### DISCUSSION TITLE
启动报错 'simbotApplicationAfterApplication' threw exception with message: Bot login failed. cause: null

### DISCUSSION THREAD

**Discussion by Lidesam** [simple-robot/Discussions#82] [打扰了]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

用的3.0.0.0-M5   ,3.0.0-RC.3 启动后就报错,甚至没有到滑块阶段.
想问下是什么原因

**Comment by ForteScarlet** [comment]
异常相关内容请通过 [issues](https://github.com/simple-robot/simpler-robot/issues/new?assignees=ForteScarlet&labels=bug%3F&template=bug-report.yml) 反馈
